semantic_conventions/trace/database.yaml

```
@@ -451,6 +451,10 @@ groups:
         requirement_level:
           conditionally_required: when performing one of the operations in this list
       - ref: user_agent.original
+        brief: 'Full user-agent string is generated by Cosmos DB SDK'
+        note: >
+          The user-agent value is generated by SDK and it contains machine information, sdk version along with few flags.
```

```suggestion
           When gateway mode is used, matches value of User-Agent HTTP header, and in case of  Direct mode <please specify>
```

Users, 3rd party SDK, or integration frameworks may write their own instrumentations, especially in languages where we don't provide Cosmos SDKs or tracing. 
So, it does not seem very useful to mention it's generated by Cosmos DB SDK, but important to explain what they should put there. 
Also, having a description of what's in user agent (`contains machine information, sdk version along with few flags`) should (if important) either be very specific so that someone can implement it or removed altogether.